sklearn/cluster/_agglomerative.py

```diff
@@ -879,7 +888,9 @@ def fit(self, X, y=None):
 
         distance_threshold = self.distance_threshold
 
-        return_distance = distance_threshold is not None
+        return_distance = (distance_threshold is not None) \
+            or self.compute_distances
```

Comment:
I think we try to avoid using `\` to break long lines. Maybe the following is more inline with the project code style:

```python
        return_distance = (
            (distance_threshold is not None) or self.compute_distances
        )
```

(Note at some point in the future we will use black hopefully :)

Reply:
Fixed by 686f884.